src/main/java/frc/robot/utils/MoPrefs.java

```diff
@@ -6,6 +6,7 @@
   static final double INTAKE_ROLLER_SETPOINT = 0.3;
   static final int CLIMBER_ENCODER_LIMIT = 10000;
   static final double SHOOTER_HOOD_SETPOINT = 15;
+  static final double STORAGE_SPEED_SETPOINT = 1.0;
```

Comment:
STORAGE_SPEED_SETPOINT seems like a slightly inappropriate name, given that setpoints always mean speed in this context. I suggest STORAGE_SETPOINT.